lib/manager/bundler/artifacts.ts

```diff
@@ -115,7 +116,7 @@ export async function updateArtifacts(
     }
     const preCommands = [
       'ruby --version',
-      `gem install bundler${bundlerVersion} --no-document`,
+      `gem install bundler${bundlerVersion}`,
```

Comment:
was this wrong?

Reply:
not required, as it is configured globally by our ruby installer